### PR TITLE
Updates to get VIRTIS camera model functional with mirror kernels

### PR DESCRIPTION
**Pull request by krlberry**
These updates allow one to rosvirtis2isis, spiceinit, and run other applications without error. The resulting camera model has good pointing results in some cases, and much less good results in others (the pointing ranges from almost perfect to offset by the entire size of the body.) There was not remaining time to investigate the details of the pointing offset (when it occurs, how bad it is, etc...)

### CODE REVIEW

File: isis/src/rosetta/apps/rosvirtis2isis/rosvirtis2isis.cpp [hunk 1]
```diff
@@ -405,25 +406,21 @@ void IsisMain ()
...
-    QString startTime = iTime(etStart).UTC(); 
-    QString stopTime = iTime(etEnd).UTC(); 
+    QString startTime = iTime(etStart-16.0).UTC(); 
+    QString stopTime = iTime(etEnd-16.0).UTC(); 
 
     SpiceChar startSclkString[50]; 
     SpiceChar endSclkString[50]; 
-    sce2s_c( (SpiceInt) -226, etStart, (SpiceInt) 50, startSclkString);
-    sce2s_c( (SpiceInt) -226, etEnd, (SpiceInt) 50, endSclkString);
+    sce2s_c( (SpiceInt) -226, etStart-16.0, (SpiceInt) 50, startSclkString);
```
Comment (krlberry) [inline isis/src/rosetta/apps/rosvirtis2isis/rosvirtis2isis.cpp]: Needs to be changed to - exposureTime()

File: isis/src/base/objs/SpiceRotation/SpiceRotation.h [hunk 2]
```diff
@@ -280,7 +280,8 @@ namespace Isis {
       enum DownsizeStatus {
         Yes,  //!< Downsize the cache
         Done, //!< Cache is downsized
-        No    //!< Do not downsize the cache
+        No,   //!< Do not downsize the cache
+        Fixed //!< Never downsize the cache
```
Comment (krlberry) [inline isis/src/base/objs/SpiceRotation/SpiceRotation.h]: Remove this. Unnecessary.

File: isis/src/base/objs/SpiceRotation/SpiceRotation.cpp [hunk 3]
```diff
@@ -2296,7 +2312,7 @@ namespace Isis {
     ktotal_c("ck", (SpiceInt *) &count);
 
     // Downsize the loaded cache
-    if ((p_source == Memcache) && p_minimizeCache == Yes) {
+    if ((p_source == Memcache) && p_minimizeCache == Yes) { // FIXME me (skip this if: minimizeCache == no)
```
Comment (krlberry) [inline isis/src/base/objs/SpiceRotation/SpiceRotation.cpp]: Remove all FIXME comments. They don't need to be fixed, actually, I just use 'FIXME' as a flag that I need to take something out before comitting. (oops)

File: isis/src/base/objs/SpiceRotation/SpiceRotation.cpp [hunk 4]
```diff
@@ -306,7 +306,9 @@ namespace Isis {
    * @param status The DownsizeStatus enumeration value.
    */
   void SpiceRotation::MinimizeCache(DownsizeStatus status) {
-    p_minimizeCache = status;
+//    if (p_minimizeCache != DownsizeStatus::Fixed) {
```
Comment (krlberry) [inline isis/src/base/objs/SpiceRotation/SpiceRotation.cpp]: All of these comments need to be removed.

**Comment by jessemapel** [comment]
@kberryUSGS Is this ready to be reviewed?

**Comment by krlberry** [comment]
@jessemapel Let me clean this up. Thank you for pinging me about it.

**Comment by jessemapel** [comment]
@kberryUSGS No rush, I was mostly just curious

**Comment by krlberry** [comment]
Needs history entries in the header files.

**Comment by Kelvinrr** [comment]
Can one of the admins verify this patch?

**Comment by krlberry** [comment]
Cleaned this up and merged in dev.

**Comment by krlberry** [comment]
This is a PR into the virtis branch and not dev. I'm just going to go ahead and merge this into the virtis branch and then open a new PR for review of the virtis branch into dev.

**Pull request merged**